tests/arch/arm/arm_interrupt/src/arm_interrupt.c

```diff
@@ -458,7 +458,7 @@ void test_arm_user_interrupt(void)
 #pragma GCC push_options
 #pragma GCC optimize("O0")
 /* Avoid compiler optimizing null pointer de-referencing. */
-void test_arm_null_pointer_exception(void)
+ZTEST(arm_interrupt, test_arm_null_pointer_exception)
```

Comment:
maybe use macro ZTEST_SKIP_IFDEF better.

Reply:
Can you explain when do I need to use this ZTEST_SKIP_IFDEF ?

Reply:
> maybe use macro ZTEST_SKIP_IFDEF better.

sorry, please ignore this comment. 
this is a condition that if one config is not enabled, then test should be skipped, maybe we need to add a new macro Z_TEST_SKIP_IFNDEF for this condition.